low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -16,22 +16,27 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com
 
 import errno
-import psutil
+import re
+import shlex
 import time
-from socket import AF_INET
 from pathlib import Path
+from socket import AF_INET
 
+import psutil
+from cortx.utils.process import SimpleProcess
+from framework.base.sspl_constants import (CPU_PATH, DEFAULT_RECOMMENDATION,
```

Comment:
this way it asks for a lot of indentation going forward if we have to add more imports,
as earlier suggested by @mariyappanp, following is the better option.
```
from framework.base.sspl_constants import (
    CPU_PATH, DEFAULT_RECOMMENDATION, HEALTH_SVC_NAME, SAS_RESOURCE_ID)
```